src/content/perf/shader.md

```diff
@@ -6,61 +6,32 @@ description: What is shader jank and how to minimize it.
 
 {% render docs/performance.md %}
 
-If the animations on your mobile app appear to be janky,
-but only on the first run,
-this is likely due to shader compilation.
-Flutter's long term solution to
-shader compilation jank is [Impeller][],
-which is the default renderer on iOS.
-You can preview Impeller on Android by passing
-`--enable-impeller` to `flutter run.
+This document describes what shader compilation jank is and how to minimize it.
+
+Shader compilation jank is the annoying stutter you sometimes see in
+animations, especially the first time you run an app. This happens because the
+app has to compile shaders while it's running.
+
+If you are using [Impeller][], Flutter's default mobile renderer, you shouldn't
```

Comment:
```suggestion
If your app is using [Impeller][], Flutter's default mobile renderer, you shouldn't
```

Reply:
Even betterer:
If your app uses [Impeller][], Flutter's default mobile renderer,
you shouldn't